project/src/components/app/app.tsx

```diff
@@ -8,27 +8,30 @@ import Player from '../../pages/player/player';
 import { AppRoute, AuthorizationStatus } from '../../const';
 import NotFound from '../../pages/not-found/not-found';
 import PrivateRoute from '../private-route/private-route';
+import AddReview from '../../pages/add-review/add-review';
 
 type AppScreenProps = {
-  filmsData : FilmType[],
-}
-
+  filmsData: FilmType[];
+};
 
-function App({filmsData}: AppScreenProps): JSX.Element {
+function App({ filmsData }: AppScreenProps): JSX.Element {
   return (
     <BrowserRouter>
       <Routes>
         <Route path={AppRoute.Root} element={<Main filmPromoData={filmsData[0]} filmsData={filmsData} />} />
-        <Route path={`${AppRoute.Film}/:id`} element={<Film filmsData={filmsData} />}/>
+        <Route path={`${AppRoute.Film}/:id`} element={<Film filmsData={filmsData} />} />
+        <Route path={`${AppRoute.Film}/:id${AppRoute.AddReview}`} element={<AddReview filmsData={filmsData} />} />
```

Comment:
Заходит? Какой урл получается? Просто вот так /:id${App... выглядит странно. Лучше
```suggestion
        <Route path={`${AppRoute.Film}/:id/${AppRoute.AddReview}`} element={<AddReview filmsData={filmsData} />} />
```